docs/dev/agent_integration.md

```diff
@@ -260,168 +353,51 @@ def test_service_check(aggregator, instance):
 {{< /code-block >}}
 
 To speed up development, use the `-m/--marker` option to run integration tests only:
-
-```
-ddev test -m integration awesome
-```
-
+   ```
+   ddev test -m integration awesome
+   ```
 Your integration is almost complete. Next, add the necessary check assets.
 
 ## Create the check assets
```

Comment:
Re-adding my comment that was marked resolved:

These assets were already created when the user ran ddev create, so I think the heading could say "Populate..." instead.

Also I think "Populate integration tile assets" or something along those lines is clearer than "check"

Reply:
addressed this by editing the last section of the agent integration doc to focus on populating the tile rather than "creating a tile", and added a warning in the create a tile doc that steps can be skipped if a file scaffolding was already created